### ISSUE TITLE
MVVM issue

### ISSUE THREAD

**Issue by gaara87** [rakshakhegde/DaggerAndroidInjector#1]
From whatever i've done and read, you have claimed that you are following the MVVM pattern, but I think its incorrect for the ViewModel to contain a reference of the View.

**Comment by rakshakhegde** [comment]
I like my approach because it's easy to write Unit Tests for it. Else I'll have a bit more logic stuck in View. This is my approach.

**Comment by RJ-Clegg** [comment]
I tend to agree with @gaara87 Here. This is not an ideal use of the MVVM paradigm. UI stuff should never touch the ViewModel.
